src/odo.ts

```diff
@@ -472,10 +473,20 @@ class OdoModel {
         return this.pathToObject.get(path);
     }
 
+    public setStringToContext(name: string, uri: Uri) {
```

Comment:
@sudhirverma What's the significance of this function?

Reply:
I am storing the context path for application to get list of services.

Reply:
For an application, there can be multiple components and services, and hence multiple context paths. How are we handling that scenario ?

Reply:
for an application, I am storing only one context path and updating the context path base on the newly created component and from that context path we can get all the services for that application.

Reply:
This is not needed.